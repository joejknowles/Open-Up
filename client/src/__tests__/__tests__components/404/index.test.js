import NotFound from '../../../components/404';

import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

const shallowRenderer = ReactTestUtils.createRenderer();

it('renders 404 page as before', () => {
  const notFound = shallowRenderer.render(<NotFound />);
  expect(notFound).toMatchSnapshot();
});
