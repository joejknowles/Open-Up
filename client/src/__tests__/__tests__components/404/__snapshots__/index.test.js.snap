exports[`test renders 404 page as before 1`] = `
<h3
  className="App">
  Page not found
</h3>
`;
